internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -220,6 +220,8 @@ func (m *metadata) remove(ctx context.Context) error {
 
 // formatCollectionName returns collection name in form <shortened_name>_<name_hash>.
 // Changing this logic will break compatibility with existing databases.
```

Comment:
This line is not needed anymore (just like in https://github.com/FerretDB/FerretDB/pull/2224/files)

Reply:
Thanks :pray: